ros2_foxglove_bridge/src/ros2_foxglove_bridge.cpp

```diff
@@ -10,6 +10,15 @@ inline bool isHiddenTopicOrService(const std::string& name) {
   }
   return name.front() == '_' || name.find("/_") != std::string::npos;
 }
+
+std::string getRosDistro() {
+  const char* rosDistro = std::getenv("ROS_DISTRO");
+  if (rosDistro == nullptr) {
+    return "unknown";
```

Comment:
I think it would be good to log a warning if that happens

Reply:
OK, will try to make that happen. Do the bridge and/or studio rely on this env var?

Reply:
Yes, Studio uses it to [determine the profile and to load known datatypes](https://github.com/foxglove/studio/blob/79b42ca3e7ad20cc225b094dd791d9e273edf14f/packages/studio-base/src/players/FoxgloveWebSocketPlayer/index.ts#L285-L304). This is e.g. necessary for publishing messages from Studio

Reply:
Also worth considering is a fatal error here. This variable should be set for good operation.